apis/python/src/tiledbsoma/soma_sparse_nd_array.py

```diff
@@ -149,12 +149,12 @@ def is_sparse(self) -> Literal[True]:
         """
         return True
 
-    # TODO
-    #    def get_nnz(self) -> wint:
-    #        """
-    #        Return the number of non-zero values in the array
-    #        """
-    #        return 999
+    @property
+    def nnz(self) -> int:
+        """
+        Return the number of stored values in the array
```

Comment:
wouldn't hurt to add a caveat that explicitly stored zeroes count toward the nnz